k8s/mongodb/mongo-ss.yaml

```diff
@@ -72,7 +72,7 @@ spec:
         resources:
           limits:
             cpu: 200m
-            memory: 768Mi
+            memory: 3.5G
```

Comment:
@krish7919 I am bumping up the RAM in this change so things don't break in the near future. We discussed that we can create/track another ticket where we can evaluate the resource consumption, minimum requirements for all the services i.e. mongodb, bigchaindb, nginx instances etc and update the pods and their configuration/scheduling accordingly.